projects/documentation/src/components/layout.ts

```diff
@@ -212,6 +221,10 @@ export class LayoutElement extends LitElement {
         this.open = !this.open;
     }
 
+    get systemOrTheme() {
+        return this.system || this.theme;
+    }
```

Comment:
This is custom to our documentation site, we should be able to use ONLY `system` being there are no consumers to support.

Reply:
Yes make sense!